src/components/Loader/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Loader';
```

Comment:
The provided code seems to be unrelated to the task description. Please ensure that you submit the correct file containing the implementation of adding and deleting todos in the React Todo App with API.